app/src/Components/Login/Login.js

```diff
@@ -24,9 +24,9 @@ const Login = () => <React.Fragment>
         </table>
       </div>
       <div style={{ 'textAlign': 'center' }}>
-        <a style={{ 'textDecoration': 'none' }} href={`${window.apiUrl}security/oauth/google`}>
+        <a style={{ 'textDecoration': 'none' }} href={`${process.env.REACT_APP_FLASK_API_HOST}/mobydq/api/v1/security/oauth/google`}>
```

Comment:
I don't like that we have the /mobydq/api/v1 code redundant, but I don't know where to store it either :/

Reply:
I agree with your point. What do you think about how this is being handled in 719aef7095ef4a1e728d4468224a935656bc6877?